### PR TITLE
Closes #64

### LINKED ISSUE #64: Can't create new campaign
screen shot 2018-01-10 at 18 04 03

### PR DESCRIPTION
**Pull request by piersb**
Not quite sure why I'm required to require it, but I seem to be...

### CODE REVIEW

**Comment by piersb** [comment]
Merges with the changes from #62 so @quarterto doesn't need to do the merge.

File: client/pages/home.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import {List} from '../components/primitives';
 import Link from '../components/link';
 import {go} from '../router';
 import formJson from '@quarterto/form-json';
+require('formdata-polyfill');
```
Comment (apaleslimghost) [inline client/pages/home.js]: this would be better in `client/pages/index.js`

File: .idea/almanac.iml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (apaleslimghost) [inline .idea/almanac.iml]: you'll want to merge master in, which has your change to add these to `.gitignore`

**Review by apaleslimghost — CHANGES_REQUESTED** [review]

File: client/pages/home.js (hunk 1 above)
Comment (apaleslimghost) [reply client/pages/home.js]: and should be `import 'formdata-polyfill';` for consistency

**Comment by apaleslimghost** [comment]
> Not quite sure why I'm required to require it, but I seem to be...

Just installing it with `npm` isn't enough to make meteor include it, it needs to be referenced somewhere in the `client` code

**Review by apaleslimghost — APPROVED** [review]

**Pull request merged**